src/cephadm/tests/test_custom_container.py

```diff
@@ -72,7 +72,10 @@ def test_get_container_envs(self):
         self.assertEqual(result, ['SECRET=password'])
 
     def test_get_container_mounts(self):
-        result = self.cc.get_container_mounts('/xyz')
+        # TODO: get_container_mounts was made private. test the private func for
+        # now. in the future update to test base class func
+        # customize_container_mounts
+        result = self.cc._get_container_mounts('/xyz')
```

Comment:
was there some difficulty causing this test change to be pushed off for now? It seems like you should just be able to pass `customize_container_mounts` an empty dict of mounts and you'll get the same thing as this call is currently, as long as `ctx.data_dir` is set.

Reply:
It's my urge to change as little as possible. You're correct, I just didn't want to go all-in on that when a smaller code change would work.